packages/headless/src/features/generated-answer/answer-api-selectors.ts

```diff
@@ -8,12 +8,22 @@ export const selectAnswerTriggerParams = createSelector(
   (state) => state.generatedAnswer.cannotAnswer,
   (state) => state.configuration.analytics.analyticsMode,
   (state) => state.search.searchAction?.actionCause,
-  (q, requestId, cannotAnswer, analyticsMode, actionCause) => ({
+  (state) => state.search.readyToGenerateAnswer,
+
+  (
+    q,
+    requestId,
+    cannotAnswer,
+    analyticsMode,
+    actionCause,
```

Comment:
This `actionCause` selector should already be updated without having to have a `readyToGenerateAnswer`